### PR TITLE
Write all APIDOCs for Module

### LINKED ISSUE #264: [Feat] Write all APIDOCs for Module
make api doc in `_apidoc.js` for all requests in the route for Module

### PR DESCRIPTION
**Pull request by Abhishekgoyal007**
## Description
Write all apidocs for module

## Fixes
#264 

## Checklist

- [x] Code follows project's style guidelines.
- [x] Changes are documented appropriately.

### CODE REVIEW

File: _apidoc.js [hunk 1]
```diff
@@ -666,6 +666,53 @@
...
+ * 
+ * @apiParam {String} moduleId The ID of the Module document to delete.
+ *
+ * @apiSuccess {String} res Success message indicating the deletion.
+ *
+ * @apiError (Error 500) DatabaseError Error message if there was an error during the deletion.
+ */
+
+/**
+ * @api {post} /module/update Update Module
```
Comment (sumitst05) [inline _apidoc.js:699]: Please include the slug `:moduleId` in the url to update module.

Comment (Abhishekgoyal007) [reply _apidoc.js]: Done !

**Review by sumitst05 — APPROVED** [review]
LGTM!

**Comment by TejasNair9977** [comment]
oh crap

**Comment by TejasNair9977** [comment]
i forgot i had to make this change literally everywhere after modifying the endpoints a while back

**Comment by TejasNair9977** [comment]
oh man

**Pull request merged**